juriscraper/opinions/united_states/state/tex.py

```diff
@@ -162,16 +163,21 @@ def parse_lower_court_info(title: str) -> tuple[str, str]:
         if match := re.search(texapp_regex, title):
             lower_court = match.group("lower_court")
             lower_court_number = title[match.end() :].split(",")[0]
```

Comment:
docstrings on this function are not correct. 

also Instead of not returning BODA id I created one in courts-db `txboda`